lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1768,6 +1768,7 @@ sub test_resultfile_list {
     # get a list of existing resultfiles
     my ($self) = @_;
     my $testresdir = $self->result_dir();
+    return undef unless $testresdir;
```

Comment:
I haven't checked how the result is used. Maybe it would be better to return an empty array ref here instead of undef?
If it used as an array ref later, it will be autovivified into one anyway.

Reply:
so `return []`?

Reply:
Yep